
barrett_hw:
  # CONTROLLERS USED IN THE EXAMPLE
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 100 
